### PR TITLE
ATenOp Support for BCEWithLogitsLoss

### PR DESCRIPTION
**Pull request by iK1D**
Add ATenOp support for BCEWithLogitsLoss. This is required by one of the customer models. The previous exported sub-graph may have data overflow compared to PyTorch's implementation, and the AUC is worse. When switch to ATenOp, the AUC is compatable to baseline.

### CODE REVIEW

**Comment by zhijxu-MS** [comment]
thanks vincent!

the code to reprocduce overflow in ort while not in torch.

tensor = torch.tensor([-7.8477,  7.3320, -7.8008,  6.6016]).cuda().half()
label = torch.tensor([0, 1, 0, 1]).cuda().half()
torch_model1 = model()
cprint(f"torch res 1 is {torch_model1(tensor, label)}", "red")
model1 = ORTModule(torch_model1)
cprint(f"ort res 1 is {model1(tensor, label)}", "red")

**Review by SherlockNoMad — APPROVED** [review]
